templates/default-dlint-yaml

```diff
@@ -0,0 +1,3 @@
+# Warns about uses of snake_case instead of camelCase
+# Enabled by default. Uncomment to disable.
```

Comment:
Side note: I wonder if we should disable this by default given how few users seem to like this hint.

Reply:
Science says [underscores are better](https://ieeexplore.ieee.org/document/5521745). [[PDF](http://www.cs.kent.edu/~jmaletic/papers/ICPC2010-CamelCaseUnderScoreClouds.pdf)] 🙄